src/pages/create/EventForm.tsx

```diff
@@ -2,14 +2,14 @@ import { useTranslation } from 'react-i18next';
 
 import { OfferType } from '@/constants/OfferType';
 import { SupportedLanguages } from '@/i18n/index';
-import { Event, isEvent } from '@/types/Event';
+import { AttendenceModes, Event, isEvent } from '@/types/Event';
```

Comment:
Maybe it's better to use `AttendanceMode` singular? 
We also use `OfferType` and not `OfferTypes`?

```suggestion
import { AttendenceMode, Event, isEvent } from '@/types/Event';
```